### PR TITLE
Clarify partial method for joins

### PR DESCRIPTION
**Pull request by notebk**
I saw this solution answered by @boutell in SO. I couldn't understand this part of the docs, so adding in a bit of context for how to use the partial method with joins.

### CODE REVIEW

File: docs/advanced-topics/advanced-pieces-topics/adding-columns.md [hunk 1]
```diff
@@ -25,6 +25,24 @@ module.exports = {
 Notice that for `sort` you specify an object exactly like what you'd pass to MongoDB's `sort()` method, or Apostrophe's `sort()` cursor filter. In particular, the actual property you sort on does not have to match the property name displayed in the column. For example, when working with people's names you might sort on `{ lastName: 1, firstName: 1 }` rather than `title`.
 :::
 
+To display the title or another field of a join in a column, include the `partial` method in the `addColumns` options:
```
Comment (abea) [inline docs/advanced-topics/advanced-pieces-topics/adding-columns.md:28]: How about "the title or another field from a join" (note the "from")?

File: docs/advanced-topics/advanced-pieces-topics/adding-columns.md [hunk 2]
```diff
@@ -25,6 +25,24 @@ module.exports = {
...
+
+```javascript
+  //Other configuration options, then...
```
Comment (abea) [inline docs/advanced-topics/advanced-pieces-topics/adding-columns.md:31]: Add a space before "Other"

**Review by abea — CHANGES_REQUESTED** [review]
Thanks for this contribution! The section after your addition explains what the `partial` option is, so please move your addition after that next code block and directly before the "It's important to note..." paragraph.

**Comment by notebk** [comment]
Thanks for the feedback! I just added in the proposed changes

**Comment by abea** [comment]
Thanks, @notebk!

**Review by abea — APPROVED** [review]

**Pull request merged**